subscribers:

  camera_reading:
    topic_front: /alphasense_driver_ros/cam4/dropped/debayered
    topic_left: /alphasense_driver_ros/cam3/dropped/debayered
    topic_right: /alphasense_driver_ros/cam5/dropped/debayered
    topic_top: /alphasense_driver_ros/cam6/dropped/debayered
    queue_size: 1
    resolution: 720*540*3

publishers:

  object_detector:
    topic: /detected_objects
    queue_size: 1
    latch: false

  bounding_boxes:
    topic: /bounding_boxes
    queue_size: 1
    latch: false

  detection_image:
    topic: /detection_image
    queue_size: 1
    latch: true

image_view:

  enable_opencv: false
